azurerm/resource_arm_storage_blob.go

```diff
@@ -53,6 +54,12 @@ func resourceArmStorageBlob() *schema.Resource {
 				Default:      0,
 				ValidateFunc: validateArmStorageBlobSize,
 			},
+			"content_type": {
+				Type:          schema.TypeString,
```

Comment:
Should do a state migration test here? Just to make sure it doesn't break anything downstream?

Reply:
I have run the new `terraform plan` after the old `terraform apply`, and it turns out nothing has been changed. I think there is not needs to do state migration here. @katbyte @tombuildsstuff @paultyng what do you think?

Reply:
related question below: where does this default value come from?

Reply:
can we also document this new field in the docs (`website/docs/r/storage_blob.html.markdown`)

Reply:
Sure. The default value comes from `Content-Type` section in https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob#request-headers-all-blob-types.